apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryJob.cs

```diff
@@ -201,6 +189,23 @@ public BigQueryJob ThrowOnAnyError()
         public Task<BigQueryJob> CancelAsync(CancelJobOptions options = null, CancellationToken cancellationToken = default(CancellationToken)) =>
             _client.CancelJobAsync(Reference, options, cancellationToken);
 
+        private TableReference GetQueryTableReference()
+        {
+            var query = Resource?.Configuration?.Query;
+            if (query == null)
+            {
+                throw new InvalidOperationException("Job doesn't represent a query");
+            }
+            ThrowOnAnyError();
+            var reference =query.DestinationTable;
```

Comment:
Whoops, thanks - will fix. I won't merge this until I've talked with the BQ team and ideally DPEs. Thanks for the review though :)